packages/platform/atoms/cal-provider/BaseCalProvider.tsx

```diff
@@ -139,3 +155,20 @@ function replaceOccurrences(input: string, replacementMap: { [key: string]: stri
     return match;
   });
 }
+
+function getTranslation(key: string, language: "en" | "fr" | "pt-BR" | "de" | "es") {
+  switch (language) {
+    case "en":
+      return enTranslations[key as enTranslationKeys];
+    case "fr":
+      return frTranslations;
```

Comment:
should be `return frTranslations[key as frTranslationKeys]`; instead of returning whole translation file